system-test/logging.ts

```diff
@@ -69,17 +69,22 @@ describe('Logging', () => {
     const oneHourAgo = new Date();
     oneHourAgo.setHours(oneHourAgo.getHours() - 1);
 
+    const twoDaysAgo = new Date();
+    twoDaysAgo.setDate(twoDaysAgo.getDate() - 2);
+
     await Promise.all([
+      deleteSinks(),
       deleteBuckets(),
       deleteDatasets(),
       deleteTopics(),
-      deleteSinks(),
     ]);
 
+    // Only delete log buckets that are at least 2 days old
+    // Fixes: https://github.com/googleapis/nodejs-logging/issues/953
     async function deleteBuckets() {
       const [buckets] = await storage.getBuckets({prefix: TESTS_PREFIX});
```

Comment:
curious why we chose 1 hour in `sinks.test.js` and  2 days in `logging.ts`?

Reply:
The sinks.test.js deletes the sink during the same test run (whereas system tests only delete resources older than 1 hour - given our CI schedule this happens 1 day later).

In practice, we'll likely delete the sinks created by sinks.test.js one day later (given our CI schedule).

Reply:
There might still be a race condition here though. I will monitor closely and revise!